src/elements/Header/_Header.js

```diff
@@ -77,7 +77,7 @@ _Header.propTypes = {
   ]),
 
   /** Color of the header. */
-  color: PropTypes.oneOf(_Header._meta.props.colors),
+  color: PropTypes.oneOf(_Header._meta.props.color),
```

Comment:
The `violent` color typo made it through because this prop type validator was pointed at the wrong prop name as well.